out/production/tekwillautomation/.gitignore

```diff
@@ -0,0 +1,23 @@
+# Compiled class file
```

Comment:
add .idea folder somewhere here, you could check if you did it right by the git status, .idea folder will not be seen there anymore, also the folder in project structure will be of grey color

Reply:
Thank you!